src/user_functions/GaussianWakeVelocityAuxFunction.C

```diff
@@ -0,0 +1,76 @@
+/*------------------------------------------------------------------------*/
```

Comment:
I don't see the usage of this class anywhere. Is a commit missing, or is this code no longer needed?